wallet/wallet.go

```diff
@@ -103,7 +103,7 @@ type Wallet struct {
 	networkBackendMu sync.Mutex
 
 	lockedOutpoints  map[wire.OutPoint]struct{}
-	lockedOutpointMu sync.Mutex
+	lockedOutpointMu sync.RWMutex
```

Comment:
I'm fine with this being a RWMutex, but last I heard you wanted to revert this back to a regular mutex.  What's the status of this?